.github/workflows/ci.yml

```diff
@@ -127,8 +139,13 @@ jobs:
       run: cabal build all -j
 
     - name: Test
+      if: matrix.test-set == 'all'
       run: cabal test all -j --test-show-details=streaming
 
+    - name: Test (NoThunks-safe tests only)
+      if: matrix.test-set == 'no-thunks-safe'
+      run: cabal test ouroboros-consensus:consensus-test ouroboros-consensus:doctest ouroboros-consensus:infra-test ouroboros-consensus:storage-test ouroboros-consensus-cardano:byron-test ouroboros-consensus-cardano:shelley-test ouroboros-consensus-diffusion:infra-test ouroboros-consensus-diffusion:mock-test ouroboros-consensus-protocol:protocol-test -j --test-show-details=streaming
```

Comment:
Why `--test-show-details=streaming` and not `direct`?

Reply:
no reason except that it's already using streaming and I don't really want it to be different